stories/object-list/object-list.stories.js

```diff
@@ -0,0 +1,1193 @@
+    </div>
+    </a>
+  </li>
+  <li role="listitem" tabindex="0" class="fd-list__item fd-object-list__item fd-list__item--link">
+  <a tabindex="0" class="fd-list__link" href="#">
+    <div class="fd-object-list__container">
+      <div class="fd-object-list__intro">
+       Optional inline text very very very long
+      </div>
+      <div class="fd-object-list__header">
+        <span class="fd-avatar fd-avatar--s fd-avatar--circle"
+         style="background-image: url('http://lorempixel.com/400/400/nature/4/')">
+        </span>
+        <div class="fd-object-list__header-left">
+         <div class="fd-object-identifier fd-object-list__object-identifier">
+          <p class="fd-object-identifier__title">
+           Fitbit Versa Smart Watch, Black Fitbit Versa Smart Watch
+          </p>
+         </div>
+        </div>
+        <div class="fd-object-list__header-right">
+          <span class="fd-object-number fd-object-number--critical fd-object-list__object-number">
+            <span class="fd-object-number__text">956.00</span>
+            <span class="fd-object-number__unit">Euro</span>
+          </span>
+        </div>
+      </div>
+      <div class="fd-object-list__content">
+        <div class="fd-object-list__row">
+          <div class="fd-object-list__row-left">
+            <div class="fd-object-list__object-attribute">
+              
+              First Attribute
+            </div>
+          </div>
+          <div class="fd-object-list__row-right">
+          <span class="fd-object-status fd-object-status--negative">
+          <i class="fd-object-status__icon sap-icon--status-negative" role="presentation"></i>
+          <span class="fd-object-status__text">Negative</span>
+         </span>
+          </div>
+        </div>
+        <div class="fd-object-list__row">
+          <div class="fd-object-list__row-left">
+            <div class="fd-object-list__object-attribute">
+              Second Attribute
+            </div>
+          </div>
+          <div class="fd-object-list__row-right">
+            <span class="fd-object-status fd-object-status--critical">
+              <span class="fd-object-status__text">Critical</span>
+            </span>
+          </div>
+        </div>
+        <div class="fd-object-list__row">
+          <div class="fd-object-list__row-left">
+            <div class="fd-object-list__object-attribute">
+              Third Attribute
+            </div>
+        </div>
+      </div>
+    </div>
+    </div>
+    </a>
+  </li>
+  </ul>
+</div>
+`;
+
+navigation.storyName = 'Navigation';
+
+navigation.parameters = {
+    docs: {
+        iframeHeight: 355,
+        storyDescription: 'An object list can display links that the users can navigate to. To display a navigation object list, add the `fd-list--navigation-object` modifier class together with `fd-list--navigation` and the `fd-list__item--link` modifier class to the list elements that contain links. All items should be navigable.'
```

Comment:
Let's change this to:

An object list can display links that the users can navigate to. To display a navigation object list, add the `fd-list--navigation-object` modifier class together with `fd-list--navigation` and the `fd-list__item--link` modifier class to the list elements that contain a link. All items should be navigable.